### PR TITLE
Added how to configure monitoring for the asb

### PR DESCRIPTION
**Pull request by bergerhoffer**
@openshift/team-documentation For peer review please.

Preview: http://file.rdu.redhat.com/~ahoffer/2019/asb-monitoring/applications/service_brokers/configuring-ansible-service-broker.html#sb-configuring-monitoring-asb_sb-configuring-asb

### CODE REVIEW

**Comment by shawn-hurley** [comment]
Looks good to me!

**Comment by bergerhoffer** [comment]
@jianzhangbjz Could you review this new addition please? Thanks!

File: modules/sb-configuring-asb-monitoring.adoc [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+//
+// * applications/service_brokers/configuring-ansible-service-broker.adoc
+
+[id="sb-configuring-monitoring-asb_{context}"]
+= Configuring monitoring for the {asb-name}
+
+In order for Prometheus to monitor the {asb-name}, you must create the following resources to grant Prometheus permission to access the namespace where the {asb-name} was installed.
+
+[NOTE]
+====
```
Comment (ahardin-rh) [inline modules/sb-configuring-asb-monitoring.adoc]: Instead of a NOTE, could this be listed as a Prerequisite?

**Comment by ahardin-rh** [comment]
@bergerhoffer Just one question. Otherwise, LGTM!

File: modules/sb-configuring-asb-monitoring.adoc (hunk 1 above)
Comment (bergerhoffer) [reply modules/sb-configuring-asb-monitoring.adoc]: @ahardin-rh you know, my note was intending to say that the procedure is assuming you installed it into X namespace, but that they really could have installed it in a different one. but you're right that generically having it installed at all is a prereq, i will definitely add that. thanks for the suggestion!

**Comment by jianzhangbjz** [comment]
@zihantang-rh Could you help review this doc? Thanks!

**Comment by zihantang-rh** [comment]
LGTM, and worked in latest version.

**Comment by bergerhoffer** [comment]
Thanks @jianzhangbjz and @zihantang-rh! Merging.

**Pull request merged**